app/src/main/java/com/henagle/www/kanban/MainActivity.java

```diff
@@ -64,6 +65,20 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         }
     };
 
+    /**
+     * This is to handle the delete button being clicked on an item
+     * @param v
+     */
+    public void deleteClickHandler(View v) {
+        RelativeLayout vwParentRow = (RelativeLayout)v.getParent();
```

Comment:
We have multiple different types of layouts used throughout our project (`ConstraintLayout`, `LinearLayout`, `RelativeLayout`). Not something this PR should address, but we should explicitly decide how we want to organize our layouts.